CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Fixed a bug where missing token would crash the api. [#1367](https://github.com/openkfw/TruBudget/issues/1367)
```

Comment:
no need for changelog entry, as the bug is not relevant for Trubudget users or operators